library/std/src/primitive_docs.rs

```diff
@@ -378,7 +378,7 @@ mod prim_char {}
 ///     returns_i64()
 /// };
 /// let is_unit = {
-///     returns_i64();
+///     returns_unit();
```

Comment:
Oh, it should be better:

```suggestion
///     returns_unit()
```

Reply:
Or, I'm wondering it's intentional to show that adding `;` to the end of `returns_i64` results `()`.

Reply:
Ah yeah, it probably is 😄